### PR TITLE
Add unit tests to "PluginAddCmd" to use testify

### PR DESCRIPTION
**Pull request by Pomyk**
#### Summary

Adds unit tests for the `mmctl plugin add` command.

Fixes https://github.com/mattermost/mattermost-server/issues/13119

### CODE REVIEW

File: commands/plugin_test.go [hunk 1]
```diff
@@ -1,14 +1,104 @@
...
+	"github.com/golang/mock/gomock"
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mmctl/printer"
 
 	"github.com/spf13/cobra"
 )
 
+func (s *MmctlUnitTestSuite) TestPluginAddCmd() {
+	s.Run("Add without args", func() {
+		err := pluginAddCmdF(s.client, &cobra.Command{}, []string{})
```
Comment (mgdelacroix) [inline commands/plugin_test.go]: This test case needs to start with `printer.Clean()`

File: commands/plugin_test.go [hunk 2]
```diff
@@ -1,14 +1,104 @@
...
+			Times(1)
+
+		err = pluginAddCmdF(s.client, &cobra.Command{}, []string{pluginName})
+		s.Require().NoError(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0], "Added plugin: "+pluginName)
+	})
+
+	s.Run("Add 1 plugin no file", func() {
+		err := pluginAddCmdF(s.client, &cobra.Command{}, []string{"non_existent_plugin"})
```
Comment (mgdelacroix) [inline commands/plugin_test.go]: This test case needs to start with `printer.Clean()`

File: commands/plugin_test.go [hunk 3]
```diff
@@ -1,14 +1,104 @@
...
+		mockError := &model.AppError{Message: "Plugin Add Error"}
+
+		for idx, arg := range args {
+			tmpFile, err := ioutil.TempFile("", "tmpPlugin")
+			s.Require().Nil(err)
+			defer os.Remove(tmpFile.Name())
+			if arg == "fail" {
+				s.client.
+					EXPECT().
+					UploadPlugin(gomock.Any()).
```
Comment (ethervoid) [inline commands/plugin_test.go]: Same here

File: commands/plugin_test.go [hunk 4]
```diff
@@ -1,14 +1,104 @@
...
+					Return(nil, &model.Response{Error: mockError}).
+					Times(1)
+			} else {
+				s.client.
+					EXPECT().
+					UploadPlugin(gomock.Any()).
```
Comment (ethervoid) [inline commands/plugin_test.go]: Same here

**Comment by ethervoid** [comment]
@Pomyk You have to fix the conflicts you have :)

**Pull request merged**